app/scripts/metamask-controller.test.js

```diff
@@ -3,7 +3,7 @@
  */
 import { cloneDeep } from 'lodash';
 import nock from 'nock';
-import { obj as createThoughStream } from 'through2';
+import { obj as createThroughStream } from 'through2';
```

Comment:
note: this is just a typo correction

Reply:
can undo this if there is preference. not necessary to fix here

Reply:
no was just a note for others